src/components/Scanner.vue

```diff
@@ -1,34 +1,40 @@
 <template>
-  <ion-toolbar>
-    <ion-buttons slot="end" @click="closeScanner()" >
-      <ion-button >
-        <ion-icon :icon="closeOutline" />
-      </ion-button>
-    </ion-buttons>  
-  </ion-toolbar>   
-  <div class="scanner">
-    <StreamBarcodeReader
-      @decode="onDecode"
-      @loaded="onLoaded"
-    />
-  </div> 
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="end" @click="closeScanner()" >
+        <ion-button>
+          <ion-icon :icon="closeOutline" />
```

Comment:
I think we need to add slot here right @azkyakhan

Reply:
I have added icon-only slot to ion-icon in ion button.